build.gradle.kts

```diff
@@ -19,12 +19,55 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
  */
 
 plugins {
+  id("com.gradle.build-scan") version "2.2.1"
   `java-gradle-plugin`
   kotlin("jvm") version "1.3.21"
   id("com.github.ben-manes.versions") version "0.21.0"
   id("com.gradle.plugin-publish") version "0.10.1"
 }
 
+fun str2bool(s: String?, v: String = "true"): Boolean {
```

Comment:
I'd have an extension method here, and at least I'd simplify with `return "true" == s` as Kotlin has an equality check operator that makes the _right_ thing.